MobileWallet/Screens/Settings/SettingsParentViewController.swift

```diff
@@ -80,4 +81,43 @@ extension SettingsParentViewController {
         separator.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor).isActive = true
         separator.heightAnchor.constraint(equalToConstant: 1).isActive = true
     }
+    
+    fileprivate func checkClipboardForBaseNode() {
```

Comment:
This is base controller which used as parent for 5 controllers. If you want to check the clipboard only from settings screen you should move it to SettingsViewController.

Reply:
@SerhiiShovkoplias thanks fixed 👍